src/NosCore.Controllers/CharacterScreen/CharNewPacketHandler.cs

```diff
@@ -41,6 +41,8 @@ public class CharNewPacketHandler : PacketHandler<CharNewPacket>, IWorldPacketHa
         private readonly IDao<CharacterDto, long> _characterDao;
         private readonly IDao<MinilandDto, Guid> _minilandDao;
 
+        public const string Nameregex =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5496257)